entries/jQuery.htmlPrefilter.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.htmlPrefilter" return="String">
+  <title>jQuery.htmlPrefilter()</title>
+  <desc>Modify and filter HTML strings passed through <a href="/category/manipulation/">jQuery manipulation methods</a>.</desc>
+  <signature>
+    <added>1.12/2.2</added>
+    <argument name="html" type="String">
```

Comment:
Actually, I think String is more correct. Strings passed to this function do not need to conform to the constraints laid out by htmlString (for instance, starting with a "<").